workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/internal/RealRenderContext.kt

```diff
@@ -70,6 +84,544 @@ internal class RealRenderContext<out PropsT, StateT, OutputT>(
     sideEffectRunner.runningSideEffect(key, sideEffect)
   }
 
+  override fun <ResultT> remember(
+    key: String,
+    resultType: KType,
+    vararg inputs: Any?,
+    calculation: () -> ResultT
+  ): ResultT {
+    checkNotFrozen()
+    return rememberStore.remember(key, resultType, inputs = inputs, calculation)
+  }
+
+  override fun eventHandler(
+    name: String,
+    update: WorkflowAction<
+      @UnsafeVariance PropsT,
+      StateT,
+      @UnsafeVariance OutputT
+      >.Updater.() -> Unit
+  ): () -> Unit {
+    val handler = { actionSink.send(action("eventHandler:$name", update)) }
```

Comment:
nit: can we make this prefix `eH:`? There is very limited room in the perfetto trace char limit.

Reply:
Done.